[2023-11-14T09:27:36.016+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-14T09:27:33.057150+00:00 [queued]>
[2023-11-14T09:27:36.035+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-14T09:27:33.057150+00:00 [queued]>
[2023-11-14T09:27:36.035+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T09:27:36.036+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-14T09:27:36.036+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T09:27:36.058+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-11-14 09:27:33.057150+00:00
[2023-11-14T09:27:36.066+0000] {standard_task_runner.py:55} INFO - Started process 435 to run task
[2023-11-14T09:27:36.071+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'is_api_available', 'manual__2023-11-14T09:27:33.057150+00:00', '--job-id', '423', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmphulgg8l4']
[2023-11-14T09:27:36.074+0000] {standard_task_runner.py:83} INFO - Job 423: Subtask is_api_available
[2023-11-14T09:27:36.154+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.is_api_available manual__2023-11-14T09:27:33.057150+00:00 [running]> on host 91f7624ceef9
[2023-11-14T09:27:36.234+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-11-14T09:27:33.057150+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-14T09:27:33.057150+00:00
[2023-11-14T09:27:36.235+0000] {http.py:120} INFO - Poking: api/
[2023-11-14T09:27:36.246+0000] {base.py:71} INFO - Using connection ID 'rand_user' for task execution.
[2023-11-14T09:27:36.248+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-11-14T09:27:36.664+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-11-14T09:27:36.686+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=is_api_available, execution_date=20231114T092733, start_date=20231114T092736, end_date=20231114T092736
[2023-11-14T09:27:36.763+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-14T09:27:36.849+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
